vuu-ui/showcase/src/examples/Apps/NewTheme.examples.tsx

```diff
@@ -131,9 +135,11 @@ const ShellWithNewTheme = () => {
 
 export const ShellWithNewThemeAndLayoutManagement = () => {
   return (
-    <LayoutManagementProvider>
-      <ShellWithNewTheme />
-    </LayoutManagementProvider>
+    <NotificationsProvider>
+      <LayoutManagementProvider>
```

Comment:
can we do this for sample app as well? (vuu-ui/sample-apps/app-vuu-example/index.tsx)

Reply:
0582ea4